ledger/src/blocktree_db.rs

```diff
@@ -576,6 +581,10 @@ impl Database {
     pub fn write(&self, batch: WriteBatch) -> Result<()> {
         self.backend.write(batch.write_batch)
     }
+
+    pub fn storage_size(&self) -> Option<u64> {
+        get_size(&self.path).ok()
```

Comment:
seems like this should never return an error based on this: 
https://docs.rs/fs_extra/0.3.0/fs_extra/dir/fn.get_size.html

I think we can unwrap with expect("") here

Reply:
ok, done